src/engine/input/key-mapping/InputEventItem.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using Godot;
+
+public class InputEventItem : Node
+{
+    [Export]
+    public NodePath ButtonPath;
+
+    internal InputActionItem AssociatedAction;
+
+    internal bool JustAdded;
+
+    private Button button;
+
+    private bool waitingForInput;
+
+    public InputEventWithModifiers AssociatedEvent { get; set; }
+
+    public void Delete()
+    {
+        AssociatedAction.AssociatedGroup.GetParent<InputGroupList>().ControlSchemeChanged();
+        AssociatedAction.Inputs.Remove(this);
+    }
+
+    public override void _Input(InputEvent @event)
```

Comment:
This seems to perform key reassinging? Please add doc on this method about that.